Build.ps1

```diff
@@ -100,6 +100,14 @@ Properties {
     # will no longer be prompted for the API key.
     $NuGetApiKey = $null
     $EncryptedApiKeyPath = "$env:LOCALAPPDATA\vscode-powershell\NuGetApiKey.clixml"
+
+    # If you do not specify the certificate thumbprint when specifying a build that
+    # includes script signing the build will use the first code signing certificate
+    # it finds in the users personal certificate store, unless a path to a PFX file
+    # is specified using the CertPfxPath parameter. The build will import the 
+    # certificate (if required), then store the thumbprint encrypted in a file, so
+    # that on subsequent signing the build will use the same certificate.
+    $CertThumbprintPath  = "$env:LOCALAPPDATA\WindowsPowerShell\CertificateThumbprint.clixml"
 }
```

Comment:
I'm not sure I like the approach of grabbing the first one it finds.  I would probably lean towards - "if you want signing, you have to specify the cert" approach.  We've also gotten bit using thumbprints.  When you renew your cert, the thumbprint is different so if the old cert is left on the machine, the build keeps getting signed by the old cert.  We use 'Subject name` along with the `signtool sign a` which finds the cert with the latest expire date.  We should probably discourage folks from using files (checking in your code-signing cert file is usually not a good idea).

BTW why does the code signing Subject Name or thumbprint need to be encrypted?  You probably just need to "store" the value and encryption (as I've implemented it is for free).  I should look at making that "store" more general purpose - like it is in Plaster.  Then we have one file with a mixture of name/value pairs where some values are non-encrypted and others are encrypted.  

BTW you could just have folks enter their cert's Subject Name here and then always find the cert with that name, sort of expiration date and take the one latest date.  Or during initial run, you could list the available code-signing certs and let the user pick one.  Just thinking out loud here.